rest/resource-server/src/docs/asciidoc/releases.adoc

```diff
@@ -240,6 +240,20 @@ include::{snippets}/should_document_get_release_attachment_bundle/curl-request.a
 ===== Example response
 include::{snippets}/should_document_get_release_attachment_bundle/http-response.adoc[]
 
+[[resources-release-external-id]]
+==== Get releases by external IDs
+
+A `GET` request will allow you to get releases by external IDs. +
```

Comment:
Perhaps, the plus sign at the end is redundant

Reply:
Removed.